queries/buildings.jinja2

```diff
@@ -73,6 +79,8 @@ SELECT
     NULL AS roof_orientation,
     "addr:housenumber" AS addr_housenumber,
     "addr:street" AS addr_street,
+    tags->'addr:floor' AS addr_floor,
+    tags->'level' as level,
```

Comment:
Sorry, I didn't make my point very well. What I meant to say was: this seems like a very rarely-used tag, that doesn't provide a lot of extra information in the vast majority of cases. If you've got some examples of large apartment blocks with too many address points then we should check to see if `level` has been defined on any of those, so we can check that it'll provide the information for the edge-cases that you want it for.